### ISSUE TITLE
Priority zero crash

### ISSUE THREAD

**Issue by Mr-FLK** [asquared31415/ONI-mods#3]
Hi!

here's the log

Priority Value Out Of Range: -200

  at UnityEngine.Debug.LogErrorFormat (System.String format, System.Object[] args) [0x00000] in <9fe2cbebfa394c8da6af069ec8cc9ef4>:0 
  at Debug.LogErrorFormat (System.String format, System.Object[] args) [0x00000] in <7110358c02a14df2a7a2562defcc57da>:0 
  at Chore.Chore..ctor_Patch0 (Chore , ChoreType , ChoreProvider , System.Boolean , System.Action`1[T] , System.Action`1[T] , System.Action`1[T] , PriorityScreen+PriorityClass , System.Int32 , System.Boolean , System.Boolean , System.Int32 , System.Boolean , ReportManager+ReportType ) [0x00000] in <4d5210689d454b148ee8bf399a4b6b43>:0 
  at Chore`1[StateMachineInstanceType]..ctor (ChoreType chore_type, IStateMachineTarget target, ChoreProvider chore_provider, System.Boolean run_until_complete, System.Action`1[T] on_complete, System.Action`1[T] on_begin, System.Action`1[T] on_end, PriorityScreen+PriorityClass master_priority_class, System.Int32 master_priority_value, System.Boolean is_preemptable, System.Boolean allow_in_context_menu, System.Int32 priority_mod, System.Boolean add_to_daily_report, ReportManager+ReportType report_type) [0x00000] in <4d5210689d454b148ee8bf399a4b6b43>:0 
  at FetchAreaChore..ctor (Chore+Precondition+Context context) [0x00000] in <4d5210689d454b148ee8bf399a4b6b43>:0 
  at FetchChore.PrepareChore (Chore+Precondition+Context& context) [0x00000] in <4d5210689d454b148ee8bf399a4b6b43>:0 
  at ChoreDriver.SetChore (Chore+Precondition+Context context) [0x00000] in <4d5210689d454b148ee8bf399a4b6b43>:0 
  at SolidTransferArm.Sim () [0x00000] in <4d5210689d454b148ee8bf399a4b6b43>:0 
  at SolidTransferArm+BatchUpdateContext.Finish () [0x00000] in <4d5210689d454b148ee8bf399a4b6b43>:0 
  at SolidTransferArm.BatchUpdate (System.Collections.Generic.List`1[T] solid_transfer_arms, System.Single time_delta) [0x00000] in <4d5210689d454b148ee8bf399a4b6b43>:0 
  at UpdateBucketWithUpdater`1[DataType].Update (System.Single dt) [0x00000] in <7110358c02a14df2a7a2562defcc57da>:0 
  at StateMachineUpdater+BucketGroup.AdvanceOneSubTick (System.Single dt) [0x00000] in <7110358c02a14df2a7a2562defcc57da>:0 
  at StateMachineUpdater.AdvanceOneSimSubTick () [0x00000] in <7110358c02a14df2a7a2562defcc57da>:0 
  at Game.SimEveryTick (System.Single dt) [0x00000] in <4d5210689d454b148ee8bf399a4b6b43>:0 
  at Game.Update () [0x00000] in <4d5210689d454b148ee8bf399a4b6b43>:0 
Build: U33-474321-S

**Comment by asquared31415** [comment]
I cannot reproduce this crash, do you have a save file or steps to cause the crash?

**Comment by DDRMANIAC007** [comment]
I also get this crash when my auto sweeper tries to supply a super computer marked as priority zero to prevent manual resupply to it.

**Comment by asquared31415** [comment]
This has been fixed in the latest version of the mod, 2.0.1.  If steam doesn't give you an update within the next few hours, try Mod Updater.

**Comment by Mr-FLK** [comment]
> I cannot reproduce this crash, do you have a save file or steps to cause the crash?

Unfortunately It didn't happen ever again. I could provide you my save but I don't think it would make any difference.
What's odd now is that priority 0 isn't working for sweeping errands: I Have several tons of 1000°C+ Igneous rock flagged as sweep with 0 prio but Dupes are still pilfering it and feeding hatches with it.
I had to use all the igneous rock to build a ludicrous amount of insulated tiles to delete the heat or my colony would get toasty pretty fast... but i feel it exploity and wanted to circumvent this with the mod  :( 

edit: 
as I recall, the bug happened when I set all my storage bins to prio 0 so my dupes wouldn't create "manual" errands since the bins are handled by an autosweeper fed by a conveyor chute.
I will try and set to 0 the bins and see if the bug happens again, I'll keep you posted.
